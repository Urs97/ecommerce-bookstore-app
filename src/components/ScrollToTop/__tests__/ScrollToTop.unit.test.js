import React from 'react';
import {shallow} from 'enzyme';
import ScrollToTop from '../ScrollToTop';

describe('ScrollToTop', () => {
    test('renders ScrollToTop', () => {
        const wrapper = shallow(
            <ScrollToTop />
        );
        expect(wrapper).toMatchSnapshot();
    });
});