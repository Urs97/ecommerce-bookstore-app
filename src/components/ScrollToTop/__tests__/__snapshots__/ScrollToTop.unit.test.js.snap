// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop renders ScrollToTop 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
